skada/utils.py

```diff
@@ -515,8 +515,9 @@ def _merge_arrays(
     return output
 
 
-def qp_solve(Q, c=None, A=None, b=None, Aeq=None, beq=None, lb=None, ub=None,
-             x0=None, tol=1e-6, max_iter=1000, verbose=False, log=False):
+def qp_solve(Q, c=None, A=None, Alb=None, Aub=None, lb=None, ub=None,
+             Aeq=None, beq=None, x0=None, tol=1e-6, max_iter=1000,
+             verbose=False, log=False):
```

Comment:
Sorry @antoinecollas, but I prefer the previous version of `qp_solve`, using a constraint Ax >= b can be done by using opposite sign factors in A and b. What do you think ?

Reply:
Besides, `lb` and `ub` should be used for constraints `lb <= x <= ub`, I think

Reply:
But maybe you are right, I am not really sure what is the best implementation here... @rflamary what do you think?

Reply:
it is more standard to have a unique A,b for inequality Ax <=b (lower not upper). I agree that we should do that (sorry @antoinecollas )

Reply:
I lke Aeq, beq and A,b for equality an inequality also if we decide to cange it I think it is easier to read from teh code